tests/bluetooth/tester/src/bttester.h

```diff
@@ -228,12 +228,15 @@ struct gap_passkey_confirm_cmd {
 	uint8_t match;
 } __packed;
 
+#define GAP_START_DIRECTED_ADV_HD	0x0001
+#define GAP_START_DIRECTED_ADV_OWN_ID	0x0002
+#define GAP_START_DIRECTED_ADV_PEER_RPA	0x0004
```

Comment:
Suggest to use BIT() for bitfields:
```suggestion
#define GAP_START_DIRECTED_ADV_HD	BIT(0)
#define GAP_START_DIRECTED_ADV_OWN_ID	BIT(1)
#define GAP_START_DIRECTED_ADV_PEER_RPA	BIT(2)
```